src/style/cell.ts

```diff
@@ -2,14 +2,14 @@ import { css } from '@linaria/core';
 
 export const cell = css`
   @layer rdg.Cell {
-    /*
-    Cannot use these because of a Chromium bug:
-    https://bugs.chromium.org/p/chromium/issues/detail?id=1326946
-    once this is fixed we can also remove "position: relative:"
-    contain: strict;
-    */
+    /* max-content does not work with size containment
+     * dynamically switching between different containment styles incurs a heavy relayout penalty
+     * Chromium bug: at odd zoom levels or subpixel positioning, layout/paint containment can make cell borders disappear
+     *   https://bugs.chromium.org/p/chromium/issues/detail?id=1326946
+     */
+    contain: style;
     position: relative; /* needed for absolute positioning to work */
-    contain: size style;
+    z-index: 0;
```

Comment:
I think I accidentally committed this in, I don't think we need it unless I'm missing something
```suggestion
```